application/comit_node/src/settings/mod.rs

```diff
@@ -110,4 +199,26 @@ mod tests {
             .is_equal_to(&Duration::from_secs(20));
     }
 
+    #[test]
+    fn config_folder_does_not_exist_will_create_folder_and_config_file() {
+        let tmp_dir = env::temp_dir();
+        let config_path = Path::join(&tmp_dir, "i_am_invincible");
+        let config_file = "default.toml";
+        if config_path.exists() {
+            if config_path.clone().join(config_file).exists() {
+                let default_config_file = config_path.clone().join(config_file);
+                fs::remove_file(default_config_file).unwrap();
+            }
+            fs::remove_dir(config_path.clone()).unwrap();
+        }
+
+        let default_settings =
+            ComitNodeSettings::create_with_default(config_path.clone(), config_file.clone());
+        let settings = ComitNodeSettings::read(config_path.join(config_file));
```

Comment:
📣 I think we should delete after to not endup with a million files in `/tmp` on our machines.

Reply:
Done